mp/src/game/shared/neo/neo_gamerules.h

```diff
@@ -94,6 +90,14 @@ class C_NEO_Player;
 
 extern ConVar neo_sv_player_restore;
 
+enum NeoGameType {
+	TDM = 0,
+	CTG,
+	VIP,
+
+	Total // Number of game types
+};
```

Comment:
As this is not an `enum class`, but rather just an `enum`. Aren't those enums could clash namespace wise? I know `NeoGameType::` seemed able to use but either make it `enum class` or keep `enum` and prefix the names: `NEO_GAME_TYPE_...` and use it directly C-style.